File: VB/InfiniteAsyncSourceMVVMSample/IssueService/UserData.vb
Namespace InfiniteAsyncSourceMVVMSample

    Public Class UserData

        Public Sub New(ByVal id As Integer, ByVal firstName As String, ByVal lastName As String)
            Me.Id = id
            Me.FirstName = firstName
            Me.LastName = lastName
        End Sub

        Public ReadOnly Property Id As Integer

        Public ReadOnly Property FirstName As String

        Public ReadOnly Property LastName As String

        Public ReadOnly Property FullName As String
            Get
                Return FirstName & " " & LastName
            End Get
        End Property
    End Class
End Namespace

File: CS/InfiniteAsyncSourceMVVMSample/IssueService/UserData.cs
using System;

namespace InfiniteAsyncSourceMVVMSample {
    public class UserData {
        public UserData(int id, string firstName, string lastName) {
            Id = id;
            FirstName = firstName;
            LastName = lastName;
        }

        public int Id { get; }
        public string FirstName { get; }
        public string LastName { get; }
        public string FullName => FirstName + " " + LastName;
    }
}
